src/components/GenreSelect.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react';
+import './GenreSelect.css';
+
+interface Props {
+	selectedGenre: string | null;
+	genres: string[];
+	onSelect: (genre: string) => void;
+}
+
+export function GenreSelect({ selectedGenre, genres, onSelect }: Props): ReactElement {
+	return <ul className="genre-list">
+		{genres.map((genre: string, index: number) =>
```

Comment:
It`s better to use parentheses if return statement contains more than 1 line

Reply:
Corrected